camelot/admin/action/list_filter.py

```diff
@@ -239,32 +244,32 @@ def value_to_string(self, filter_value, admin):
         raise NotImplementedError
     
     @property
-    def name(self):
-        return self._name
+    def key(self):
+        return self._key
     
     def get_verbose_name(self):
         if self._verbose_name is not None:
             return self._verbose_name
-        return ugettext(self.name.replace(u'_', u' ').capitalize())
+        return ugettext(self.key.replace(u'_', u' ').capitalize())
 
 class FieldFilter(AbstractFilterStrategy):
     """
     Abstract interface for defining a column-based filter clause on a queryable attribute of an entity, as part of that entity admin's query.
     Implementations of this interface should define it's python type, which will be asserted to match with that of the set attribute.
     """
-    
+
     attribute = None
-    operators = []
 
-    def __init__(self, attribute, where=None, name=None, verbose_name=None):
+    def __init__(self, attribute, where=None, key=None, verbose_name=None, choices=None):

```

Comment:
wat is de betekenis van choices hier ?

Reply:
Dat geeft aan of het attribuut waarop gefilterd wordt choices heeft gedefinieerd. Dan worden die namelijk ook toegepast op de velden voor de filterwaardes, en dat heeft ook implicaties op de operatoren die beschikbaar zijn. Bij een enumeration kolom is de like operator bv. niet nodig, daar kan door die choices altijd de eq of neq gebruikt worden.

Reply:
Maar dat ben ik in mijn volgende PR aan het uitwerken, en gaat mss nog herbekeken worden. Dus die parameter is sowieso TBD.

Reply:
Bij deze, in https://github.com/v-finance/camelot/pull/252 is deze enkel nog gedefinieerd voor de nieuw ChoicesFilter strategy.